client/src/components/GuestMenu.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import ThankYou from './ThankYou.jsx';
+import Menu from '../../../server/user_db_routes/testgetrestaurant_1.js';
+import io from 'socket.io-client';
+
+const url = 'http://localhost:8080';
+const socket = io(url);
+// Needs to get menu from server
+// Needs to render menu
+// Local hook for this guest's order
+// submit button to send order to server
+//    Should inform host using socket.io
+// For now just pass menu in as prop
+
+const GuestMenu = ({ joinName }) => {
+  // state declare here
+  const [currentOrder, setCurrentOrder] = useState([]);
+  const [submitted, setSubmitted] = useState(false);
+
+  // Map Menu (prop or import for now) items
+  const menuItems = Menu.result.menus[0].menu_sections.map((section, i) => {
+    return (
+      <details key={i}>
+        <summary> {section.section_name} </summary>
+        <ul>
+        {section.menu_items.map((item, j) => {
+          return (
+            <li key={j}>
+              {item.name} ${item.price}
+              <button type='button' onClick={() => { addItem(item); }}> + </button>
```

Comment:
> Not 100% sure, but I don't think you need `{}` after `=>` unless you have more than one expression. If that works you can probably also lose the semicolon.

Normally there are two format, if you want to ignore return then it can be `(args) => (<div>...</div>)`, otherwise can use full format `(args) => {return <div>...</div>}`, the advantage of second one is you can add other other code lines before return.